iib/workers/tasks/fbc_utils.py

```diff
@@ -81,11 +86,15 @@ def merge_catalogs_dirs(src_config: str, dest_config: str):
     :param str src_config: source config directory
     :param str dest_config: destination config directory
     """
+    from iib.workers.tasks.opm_operations import opm_validate
+
     for conf_dir in (src_config, dest_config):
         if not os.path.isdir(conf_dir):
             msg = f"config directory does not exist: {conf_dir}"
             log.error(msg)
             raise IIBError(msg)
+        enforce_json_config_dir(conf_dir)
+        opm_validate(conf_dir)
```

Comment:
If we are adding opm validate, how about we add validate after opm migrate every time before we generate the dockerfile or build the index image? this will add more robustness to our built index image

Reply:
@lipoja Ah yes, I read your comment thoroughly this time :)

Reply:
I've added a call to `opm_validate` after the migrate command under `opm_migrate`, please let me know whether it's what you're expecting.

Reply:
OK, my assumption was that it can validate SQLite as well but it can't. 
Then I guess it should be enough to have it in this function. 
What about to move it at the bottom of this function? So we do not call it twice but validate the actual result of merge?

Reply:
That's a great idea!

I'll adjust it

Reply:
@lipoja added on 7c0b5189b6f07208ed6cdb7cccd33e0760234731